README.md

```diff
@@ -23,11 +20,6 @@
 
 ## Authors
 
-👤 **Suzana marsela**
-
-- GitHub: [@githubhandle](https://github.com/Suzi216)
```

Comment:
- [OPTIONAL] Kindly make the live demo a link. There is no space between [] and ()